Sources/KeyboardShortcuts/RecorderCocoa.swift

```diff
@@ -28,7 +28,15 @@ extension KeyboardShortcuts {
 	public final class RecorderCocoa: NSSearchField, NSSearchFieldDelegate {
 		private let minimumWidth: Double = 130
 		private var eventMonitor: LocalEventMonitor?
-		private let shortcutName: Name
+		public var shortcutName: Name {
+			didSet {
```

Comment:
`didSet` is not called when we assign in an initializer. So we need to move the logic here to a private method and also call that method in the init.